CMakeLists.txt

```diff
@@ -158,6 +154,23 @@ configure_package_config_file(QUICKConfig.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/Q
 
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/QUICKConfigVersion.cmake ${CMAKE_CURRENT_BINARY_DIR}/QUICKConfig.cmake DESTINATION ${CMAKE_PACKAGE_CONFIG_DIR})
 
+#--------------------------------------------------------------	
+# Additional files
+
+# install modules
+if(NOT INSIDE_AMBER)
+	install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/amber-modules DESTINATION include)
+endif()
+
+# install data in basis dir
+install(DIRECTORY basis DESTINATION .)
+
```

Comment:
Aren't `basis` and `test` already existent?

Reply:
Yes, but these need to be copied to the install location so that tests can be run there.

Reply:
Sorry but aren't they already in the current directory (`.`)? Why to set the `DESTINATION` to `.` again?

Reply:
The DIRECTORY argument to install() doesn't work with empty strings IIRC.  So you have to give it some kind of file path to work with.

Reply:
I must misunderstand something here. I think `basis` and `test` are already in the current directory as source codes before running CMAKE, while the `install` command here seems to copy them from current directory to current directory. Please correct me.

Reply:
It seems that `.` here stands for the `./install` directory but not the current directory?

Reply:
Is `.` ${CMAKE_INSTALL_PREFIX} ?

Reply:
yep